processor/src/test/java/org/mapstruct/ap/test/template/InheritConfigurationTest.java

```diff
@@ -53,7 +53,7 @@ public void shouldInheritConfigurationSingleCandidates() {
         assertThat( createdTarget ).isNotNull();
         assertThat( createdTarget.getStringPropY() ).isEqualTo( "1" );
         assertThat( createdTarget.getIntegerPropY() ).isEqualTo( 2 );
-        assertThat( createdTarget.getNestedResultProp() ).isEqualTo( "nested");
+        assertThat( createdTarget.getNestedResultProp() ).isEqualTo( "nested" );
```

Comment:
Hum, I thought we had a CS rule which should have discovered that long time ago. But apparently we don't...